herzog/parser.py

```diff
@@ -67,7 +67,7 @@ def __iter__(self):
         except StopIteration:
             pass
 
-    def rewind(self):
+    def rewind(self) -> None:
```

Comment:
Please omit type hints for functions or methods returning `None`. This should be applied through this PR.